### PR TITLE
Components: update the beginUpdate and endUpdate (T1175998)

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: api-reference/10 UI Components/Component/3 Methods/beginUpdate().md [hunk 1]
```diff
@@ -6,7 +6,7 @@ id: Component.beginUpdate()
 Prevents the UI component from refreshing until the [endUpdate()](/api-reference/10%20UI%20Components/Component/3%20Methods/endUpdate().md '{basewidgetpath}/Methods/#endUpdate') method is called.
```
Comment (VasilyStrelyaev) [inline api-reference/10 UI Components/Component/3 Methods/beginUpdate().md]: Looks like now the short description contradicts the updated passage.

Can we also update the short description to say that the method, for instance, *"Postpones renders that can negatively affect performance until the `endUpdate()` method is called."*

P.S. I thought about "*reduces the number of renders until the `endUpdate()` method is called*" for the short description at first, but that would mean that I could possibly call `beginUpdate` without `endUpdate` to enter a sort of "efficient mode" with reduced renders. This is not true - I still need `endUpdate` to ensure that all the necessary renders happen.

Comment (sergepilipchuk) [reply api-reference/10 UI Components/Component/3 Methods/beginUpdate().md]: Ok, thanks. I considered it!

**Review by VasilyStrelyaev — APPROVED** [review]

File: api-reference/10 UI Components/Component/3 Methods/beginUpdate().md [hunk 2]
```diff
@@ -3,10 +3,10 @@ id: Component.beginUpdate()
 ---
 ---
 ##### shortDescription
-Prevents the UI component from refreshing until the [endUpdate()](/api-reference/10%20UI%20Components/Component/3%20Methods/endUpdate().md '{basewidgetpath}/Methods/#endUpdate') method is called.
+Postpones rendering that can negatively affect performance until the [endUpdate()]({basewidgetpath}/Methods/#endUpdate) method is called.
 
 ---
-The **beginUpdate()** and **endUpdate()** methods prevent the UI component from excessive updates when you are changing multiple UI component settings at once. After the **beginUpdate()** method is called, the UI component does not update its UI until the **endUpdate()** method is called.
+The **beginUpdate()** and **endUpdate()** methods reduce the number of renders in cases where extra renders can negatively affect performance.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/Component/3 Methods/beginUpdate().md]:
```suggestion
The **beginUpdate()** and **endUpdate()** methods reduce the number of renders in cases where extra rendering can negatively affect performance.
```

File: api-reference/10 UI Components/Component/3 Methods/endUpdate().md [hunk 3]
```diff
@@ -6,7 +6,7 @@ id: Component.endUpdate()
 Refreshes the UI component after a call of the [beginUpdate()](/api-reference/10%20UI%20Components/Component/3%20Methods/beginUpdate().md '{basewidgetpath}/Methods/#beginUpdate') method.
 
 ---
-Main article: [beginUpdate()](/api-reference/10%20UI%20Components/Component/3%20Methods/beginUpdate().md '{basewidgetpath}/Methods/#beginUpdate')
+The **beginUpdate()** and **endUpdate()** methods reduce the number of renders in cases where extra renders can negatively affect performance.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/Component/3 Methods/endUpdate().md]:
```suggestion
The **beginUpdate()** and **endUpdate()** methods reduce the number of renders in cases where extra rendering can negatively affect performance.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**